cli/bpmetadata/proto/bpmetadata.proto

```diff
@@ -2,87 +2,99 @@ syntax = "proto3";
 
 package google.cloud.config.bpmetadata;
 
-import "google/protobuf/struct.proto";
 import "bpmetadata_ui.proto";
+import "google/protobuf/struct.proto";
 
 // TODO: update copybara configuration for go to java package transformation
 option go_package = "github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli/bpmetadata";
 
 // BlueprintMetadata defines the overall structure for blueprint metadata.
 // The cli command i.e. `cft blueprint metadata` attempts at auto-generating
 // metadata if the blueprint is structured based on the TF blueprint template
-// i.e. https://github.com/terraform-google-modules/terraform-google-module-template
+// i.e.
+// https://github.com/terraform-google-modules/terraform-google-module-template
 // All fields within BlueprintMetadata and its children are denoted as:
 // - Gen: auto-generated - <data source>
 // - Gen: manually-authored
-// - Gen: partial (contains nested messages that can include both auto-generated and manually authored)
+// - Gen: partial (contains nested messages that can include both auto-generated
+// and manually authored)
 message BlueprintMetadata {
   // APIVersion is the apiVersion field of a metadata file
   // Gen: auto-generated
-  string api_version = 1; // @gotags: json:"apiVersion,omitempty" yaml:"apiVersion,omitempty"
+  string api_version =
+      1;  // @gotags: json:"apiVersion,omitempty" yaml:"apiVersion,omitempty"
```

Comment:
These changes look like formatting changes (maybe IDE?). Could we revert these and follow up in a different PR if it's only cosmetic?

Reply:
Done